### DISCUSSION TITLE
Add Transifex CLI as a package

### DISCUSSION THREAD

**Discussion by pauby** [chocolatey-community/chocolatey-packages#2133] [Ideas]
Can we add the [Transifex CLI](https://github.com/transifex/cli) in here? This is the new 'Transifex Client' that we currently have in this repository, so it makes sense to add it.

The Transifex Client has been deprecated in favour of this, so there is work to do there too.

**Comment by AdmiringWorm** [comment]
As the Transifex Client is already recommended to not be used by the developers, and to instead use Transifex CLI this makes sense.

Approving this and creating an issue based on it.
